dispatcher/nm-dispatcher.c

```diff
@@ -650,8 +664,9 @@ script_must_wait (const char *path)
 
 		dir = g_path_get_dirname (link);
 		real = realpath (dir, NULL);
-
-		if (real && !strcmp (real, NMD_SCRIPT_DIR_NO_WAIT))
+		if (real)
+			basename = g_path_get_basename (real);
+		if (basename && !strcmp (basename, "no-wait.d"))
```

Comment:
I think this should be just `if (real && g_str_has_suffix (real, "/no-wait.d"))`

Reply:
Okay.